README.md

```diff
@@ -42,14 +42,12 @@ Get started with the benchmarks:
 ## Benchmark Results
 
 <!-- PERFORMANCE_RESULTS_START -->
-
-| Server           | Requests/sec | Latency (ms) |
-| ---------------- | -----------: | -----------: |
-| [Tailcall]       |   `2,890.68` |      `34.69` |
-| [Gqlgen]         |     `935.00` |     `115.73` |
-| [Apollo GraphQL] |     `793.37` |     `128.22` |
-| [Netflix DGS]    |     `597.39` |     `191.85` |
-
+| Server | Requests/sec | Latency (ms) |
+|--------|--------------|--------------|
+| apollo | 715.153 | 144.277 |
+| netflixdgs | 555.187 | 209.383 |
+| gqlgen | 840.553 | 129.223 |
+| tailcall | 2645.06 | 37.8433 |
```

Comment:
Rather than push the changes yourself, it's better if you pull the changes from the main because it has an automated bot that will run the benchmark for the PRs

Reply:
Yes, @ologbonowiwi. I was actually testing the changes out on Codespaces and mistakenly pushed the changes. 🔰 on codespaces 😅. Fixing it.